3542/1/lib/repo.rb

```diff
@@ -0,0 +1,71 @@
+require 'httparty'
+require 'nokogiri'
+require 'open-uri'
+
+#:reek:TooManyInstanceVariables
+#:reek:InstanceVariableAssumption
+class Repo
+  URI = 'https://api.github.com/search/repositories'.freeze
+
+  attr_reader :name, :used_by, :watched_by_amount, :stars, :forks, :contributors_amount, :issues
+
+  def initialize(gem_name)
+    repository gem_name
```

Comment:
I have no idea what does `repository` method unless I take a look into its definition
1 its name should be verb
2 if you have `@repo` defined in the `repository` name, why can't you define this instance variables below (`@html`, `@name` ... ) in that method as well?